perf-dash.k8s.io/README.md

```diff
@@ -1,8 +1,19 @@
 To bootstrap [Perfdash](https://github.com/kubernetes/perf-tests/tree/master/perfdash):
 
 ```bash
-kubectl apply -f certificate.yaml
-kubectl apply -f deployment.yaml
-kubectl apply -f ingress.yaml
-kubectl apply -f service.yaml
+PERFDASH_REPO_URL="https://raw.githubusercontent.com/kubernetes/perf-tests/master/perfdash"
+PERFDSAH_MANIFEST_URL_DEPLOYMENT="${PERFDASH_REPO_URL}/deployment.yaml"
+PERFDASH_MANIFEST_URL_SERVICE="${PERFDASH_REPO_URL}/perfdash-service.yaml"
+PERFDASH_NAMESPACE="perfdash"
+
+kubectl apply -n "$PERFDASH_NAMESPACE" -f "$PERFDSAH_MANIFEST_DEPLOYMENT"
```

Comment:
DSAH -> DASH

Reply:
Thank you for catching that and sorry for mistake 😳

Reply:
Done